submissions/xT0PERx/html-movie-seat-booking/main.css

```diff
@@ -90,18 +86,17 @@ html {
 .placement {
   display: grid;
   grid-template-rows: repeat(6, [row] 1fr);
```

Comment:
You use `display: grid` for column layout. It's a good choice if you need to control the height of inner elements.
If not, will be enough simple `display: block` for inner elements to increase their width to 100% and positioned vertically.)